src/components/ui/IconButton.js

```diff
@@ -2,15 +2,15 @@ import PropTypes from 'prop-types'
 import { css } from 'styled-components'
 import styled from 'styled-components/macro'
 
-const StyledIconButton = styled.button`
-  cursor: pointer;
+import ResetButton from './ResetButton'
+
+const StyledIconButton = styled(ResetButton)`
```

Comment:
You can remove cursor:pointer from here too

Reply:
Oh sorry if you did already!

Reply:
uh i think i did cuz i searched for 'pointer' in the file and could not find